Guides/Development.md

```diff
@@ -30,6 +30,8 @@ Run `swift build` or simply `make` in the project's root directory.
 
 If you add symbols you may need to run `make exports` which will generate [Sources/MongoSwiftSync/Exports.swift](Sources/MongoSwiftSync/Exports.swift). This makes symbols declared in `MongoSwift` available to importers of `MongoSwiftSync`.
 
+Note: Do not run `make clean`.
```

Comment:
Is this true? Do we need to update make clean to make it work? I just remember it messing things up for me

Reply:
okay, deleted